src/audio_backend/alsa.rs

```diff
@@ -18,10 +18,17 @@ impl Sink for AlsaSink {
     fn start(&mut self) -> io::Result<()> {
         if self.0.is_some() {
```

Comment:
This is obviously not your change, but can you make this an `is_none` and get rid of the else block

Reply:
okee, I'll do